### ISSUE TITLE
failed: doc generation failure in release-2.5

### ISSUE THREAD

**Issue by ignasi35** [akka/akka#28708]
https://jenkins.akka.io:8498/job/akka-release-2.5/201/console

```
[error] /localhome/jenkinsakka/workspace/akka-release-2.5/akka-actor/src/main/scala-2.13-/akka/util/ByteString.scala:376:61: ambiguous reference to overloaded definition,
[error] both method position in class ByteBuffer of type (x$1: Int)java.nio.ByteBuffer
[error] and  method position in class Buffer of type ()Int
[error] match expected type ?
[error]           else ByteString1(dst.array, dst.arrayOffset + dst.position, dst.remaining)
[error]                                                             ^
[error] /localhome/jenkinsakka/workspace/akka-release-2.5/akka-actor/src/main/scala-2.13-/akka/util/ByteString.scala:387:61: ambiguous reference to overloaded definition,
[error] both method position in class ByteBuffer of type (x$1: Int)java.nio.ByteBuffer
[error] and  method position in class Buffer of type ()Int
[error] match expected type ?
[error]           else ByteString1(dst.array, dst.arrayOffset + dst.position, dst.remaining)
[error]                                                             ^
[info] No documentation generated with unsuccessful compiler run
[error] two errors found
```

**Comment by ignasi35** [comment]
Also affects `akka-publish-2.5-nightly`: https://jenkins.akka.io:8498/job/akka-publish-2.5-nightly/73/console

**Comment by ignasi35** [comment]
And also `akka-publish-nightly` https://jenkins.akka.io:8498/job/akka-publish-nightly/1334/consoleFull

**Comment by patriknw** [comment]
This was fixed yesterday in master, https://github.com/akka/akka/pull/28704
Could you backport to release-2.5 branch?

**Comment by ignasi35** [comment]
> This was fixed yesterday in master, #28704

See #28709 😒

**Comment by ignasi35** [comment]
When #28709 is merged I'll backport both to `release-2.5` at once.

**Comment by ignasi35** [comment]
backported `release-2.5` in 20b973e138d91dcbfd99ca0e968dbf8e128a519b
